### DISCUSSION TITLE
请问知网作者对应code怎么获取呢？

### DISCUSSION THREAD

**Discussion by yzy-lex** [DIYgod/RSSHub#12381] [Q&A]
### 路由地址

```routes
NOROUTE
```

### 完整路由地址

```fullroutes
NOROUTE
```

### 相关文档

https://docs.rsshub.app/journal.html#zhong-guo-zhi-wang

### 预期是什么？

获取知网作者对应code。

### 实际发生了什么？

![CleanShot 2023-04-23 at 23 46 44@2x](https://user-images.githubusercontent.com/50101175/233849896-532033b1-6365-43ba-9368-8217dabd1d2f.png)
请问教程里面所写的作者对应code怎么获取呢？
https://kns.cnki.net/kcms2/author/detail?v=3uoqIhG8C45UgIk_lOaz1xHQ_yjED3Fb001z6mbwMVT_z7xM9vSKN5sp6EOIXeXOl6tohQQRNLimseHUant9gvc9lyCX-4DUAOx-1zmVCw85J69cksj3EzEjEtJAB4Le&uniplatform=NZKPT

### 部署

RSSHub 演示 (https://rsshub.app)

### 额外信息

```shell
无
```

### 这不是重复的 issue

- [X] 我已经搜索了[现有 issue](https://github.com/DIYgod/RSSHub/issues)，以确保该错误尚未被报告。

**Comment by Derekmini** [comment]
https://github.com/DIYgod/RSSHub/pull/11269#issuecomment-1528775936

但是目前该路由已经失效了
